.github/workflows/build_and_test.yml

```diff
@@ -206,12 +206,12 @@ jobs:
       run: |
         mkdir dist
         for w in `find wheelhouse/ -type f -name "*.whl"` ; do cp $w dist/ ; done
-        #- name: Publish wheels
-        #  uses: pypa/gh-action-pypi-publish@release/v1
-        #  with:
-        #    user: __token__
-        #    password: ${{ secrets.PYPI_EXTENSIONS_API_TOKEN }}
-        #    verbose: true
+    - name: Publish wheels
+      uses: pypa/gh-action-pypi-publish@release/v1
+      with:
+        user: __token__
+        password: ${{ secrets.PYPI_PYTKET_QUANTINUUM_API_TOKEN }}
+        verbose: true
 
```

Comment:
Can we combine the Linux, MacOS and Windows jobs into a single job using `matrix` for the OS? (For an example, see e.g. the `test_libs.yml` in the tket repo.) (We can still perform steps like the mypy check conditionally on the OS.)